github/repos_deployments.go

```diff
@@ -176,6 +175,26 @@ func (s *RepositoriesService) ListDeploymentStatuses(owner, repo string, deploym
 	return *statuses, resp, err
 }
 
+// GetDeploymentStatus returns a single deployment status of a repository.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/deployments/#get-a-single-deployment-status
+func (s *RepositoriesService) GetDeploymentStatus(owner, repo string, deploymentID int, deploymentStatusID int) (*DeploymentStatus, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/deployments/%v/statuses/%v", owner, repo, deploymentID, deploymentStatusID)
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
This endpoint is documented at https://developer.github.com/v3/repos/deployments/#get-a-single-deployment-status, and it says there:

![image](https://cloud.githubusercontent.com/assets/1924134/21151200/97fc10b4-c12f-11e6-9f3b-2e396225b92b.png)

Are you missing the custom accept header here?

```Go
// TODO: remove custom Accept header when deployment support fully launches
req.Header.Set("Accept", mediaTypeDeploymentStatusPreview)
```